sdk/queue/README.md

```diff
@@ -0,0 +1,9 @@
+Vault SDK - Queue
+=================
+
+The `queue` package provides Vault plugins with a Priority Queue. It can be used
+as an in-memory list of `queue.Item` sorted by their `priority`, and offers
+methods to find or remove items by their key. Internally it
+uses `container/heap`; see [Example Priority
+Queue](https://golang.org/pkg/container/heap/#example__priorityQueue)
```

Comment:
A agree that a doc comment over the PriorityQueue struct would be better, or a doc.go file

Reply:
I added package level documentation in a previous commit